EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionTreeRows/ParameterBaseRowViewModel.cs

```diff
@@ -344,6 +344,19 @@ protected override void UpdateThingStatus()
             this.ThingStatus = new ThingStatus(this.Thing);
         }
 
+        /// <summary>
+        /// Update the model code property of itself and all contained rows recursively
+        /// </summary>
+        public void UpdateModelCode()
+        {
+            this.ModelCode = this.Thing.ModelCode();
+            foreach (var containedRow in this.ContainedRows)
```

Comment:
Maybe an idea to create a method  at IRowViewModelBase level that handles the repeating code? Then we cannot forget this and in the rare case that there is an IRowViewModel in the visual tree that doesn't implement IHaveModelCode the traversal chain is not broken.

Reply:
I discussed with Alex, and there are enough differences now between Element definition and product tree implementations and different row types that doesn't make sense to abstract to higher level. He said not to do it for now.